### ISSUE TITLE
A warning is issued when RegExp type value is passed to the query prop.

### ISSUE THREAD

**Issue by takeaship** [kawamataryo/vue-word-highlighter#574]
The issued warning is:

```
[vue warn]: Invalid prop: type check failed for prop "query". Expected String, Object, got RegExp
```

The suspected code is:
https://github.com/kawamataryo/vue-word-highlighter/blob/main/vue-word-highlighter/src/components/index.ts#L12

The specified type `[String, Object as () => RegExp]` seemed to be recognized as String and Object type.

**Comment by takeaship** [comment]
I'm trying to fix this issue.

**Comment by kawamataryo** [comment]
This is fixed in v1.1.2.
Thank you for your contribution.
